### DISCUSSION TITLE
What does `conformZodMessage.VALIDATION_SKIPPED` do that isn't achieved by returning early from the function?

### DISCUSSION THREAD

**Discussion by james-tindal** [edmundhung/conform#740] [General]
What does `conformZodMessage.VALIDATION_SKIPPED` do that isn't achieved by returning early from the function?

**Comment by edmundhung** [comment]
Since the zod schema will always validate the whole form data even though you are just typing on a particular field. The message tells Conform that it is safe to use the previous error instead of running some expensive validation again.

If you return early, zod will simply say the field is valid even though it might not.
